src/shims/unix/linux/eventfd.rs

```diff
@@ -1,14 +1,20 @@
 //! Linux `eventfd` implementation.
-//! Currently just a stub.
 use std::io;
+use std::io::{Error, ErrorKind};
 
 use rustc_target::abi::Endian;
 
 use crate::shims::unix::*;
-use crate::*;
+use crate::{concurrency::VClock, *};
 
 use self::shims::unix::fd::FileDescriptor;
 
+/// Minimum size of u8 array to hold u64 value.
+const U64_MIN_ARRAY_SIZE: usize = 8;
```

Comment:
I guess, all hosts and targets have the same size. Feels weird tho

Reply:
If it was target-dependent it surely couldn't be a `const`.

The "min" in the name is also odd. a u64 value is *exactly* 8 bytes, not "at least" 8 bytes.

Reply:
It's the minimum size the buffer must be to not return EINVAL

Reply:
This const is mainly used for checking if the size of slice passed in for ``read`` and ``write``. ``min`` is mentioned because it is possible to pass in an array of size 9 (this is demonstrated in the test) for ``read`` and ``write``. But it is indeed confusing and can be phrased better.

Reply:
We will always read exactly 8 bytes, right? If we are given more, we still just read 8. So in that sense this is the exact size every read must have, not the minimum size. Larger buffers are simply ignored.

Reply:
Yes correct.